examples/notebooks/get-started/learn_beam_basics_by_doing.ipynb

```diff
@@ -0,0 +1,1070 @@
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ISm7PF9wE7Ts"
+      },
+      "source": [
+        "## 1.5 Writing to Text File\n",
+        "\n",
+        "Now, how do we know if we did it correctly? Let's take a look at the text file you just read.\n",
+        "\n",
+        "You may have noticed that we can't simply `print` the output `PCollection` to see the elements. In Beam, you can __NOT__ access the elements from a `PCollection` directly like a Python list.\n",
+        "\n",
+        "This is because, depending on the runner,\n",
+        "the `PCollection` elements might live in multiple worker machines.\n",
+        "\n",
+        "However, we can see our output `PCollection` by using a [`WriteToText`](https://beam.apache.org/releases/pydoc/2.27.0/apache_beam.io.textio.html#apache_beam.io.textio.WriteToText) transform to turn our `str` elements into a `txt` file (or another file type of your choosing) and then running a command to show the head of our output file."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "iz73k9d1IUGw"
+      },
+      "source": [
+        "> `beam.io.WriteToText` takes a _file path prefix_ as an input, and it writes the all `str` elements into one or more files with filenames starting with that prefix.\n",
+        "> You can optionally pass a `file_name_suffix` as well, usually used for the file extension.\n",
+        "> Each element goes into its own line in the output files."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "D80HrbeKFxCv"
+      },
+      "source": [
+        "Now, you can try it. Save the results to a file path prefix `'output'` and make the file_name_suffix `'.txt'`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "v6RvVVINKOt9"
+      },
+      "outputs": [],
+      "source": [
+        "#@title Edit This Code\n",
+        "inputs_pattern = 'data/SMSSpamCollection'\n",
+        "\n",
+        "# Remember: | is the apply function in Beam in Python\n",
+        "outputs = (\n",
+        "    pipeline\n",
+        "      | 'Take in Dataset' >> beam.io.ReadFromText(inputs_pattern)\n",
+        "      | 'Write results' >> beam.io.WriteToText(..., file_name_suffix = ...)\n",
+        "      # To see the results\n",
+        "      | 'Print results' >> beam.Map(print)\n",
+        ")\n",
+        "\n",
+        "# To run the pipeline\n",
+        "pipeline.run()\n",
+        "\n",
+        "# The command used to view your output txt file.\n",
+        "# If you choose to save the file path prefix to a different location or change the file type,\n",
+        "# you have to update this command as well.\n",
+        "! head output*.txt"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SjphdAsmOQlh"
+      },
+      "source": [
+        "> *`!` means \"run\" for Apache Beam*"
```

Comment:
not sure I follow this.

Reply:
Removed. Tried to emphasize what run means.